erpnext/accounts/report/financial_statements.py

```diff
@@ -18,7 +18,8 @@
 from six import itervalues
 from erpnext.accounts.doctype.accounting_dimension.accounting_dimension import get_accounting_dimensions, get_dimension_with_children
 
-def get_period_list(from_custom_date, to_custom_date, periodicity, accumulated_values=False, 
+non_zero_parent_accounts = {}
```

Comment:
Is there a need for a global variable, it is used at just one place?

Reply:
No need for this at all, I had added it to test someting, don't need it. Forgot to remove it I guess.